client/src/components/calender.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import BigCalendar from 'react-big-calendar';
+import 'react-big-calendar/lib/css/react-big-calendar.css'
```

Comment:
Big Calendar renders the calendar component. It comes with native methods to click on an event or empty time slot which is why we chose to use it for this.